users/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .custom_user import CustomUser
```

Comment:
Probably this import is redundant due to the [get_user_model](https://docs.djangoproject.com/en/2.1/topics/auth/customizing/#django.contrib.auth.get_user_model) helper